### DISCUSSION TITLE
JSBSIM Compile & Build / libtool not found

### DISCUSSION THREAD

**Discussion by doughodson** [JSBSim-Team/jsbsim#464] [Support requests]
I just tried to compile JSBSIM 1.1.6 using configure.... it doesn't locate libtool on my system, and indicates that it needs to be installed (or newer) -- I'm wondering if what I have is too new, and it not being located properly. I ended up installing and using cmake, which worked.

BTW, I'm glad to see interfaces to Julia!!! Thanks,
Doug

**Comment by bcoconni** [comment]
JSBSim no longer uses GNU's autotools to build. We are now using [CMake](https://cmake.org).
Please make sure CMake is installed on your platform and try running the following commands from JSBSim root folder:
```bash
> mkdir build
> cd build
> cmake ..
> make
```
Would you encounter any problem in the process, don't hesitate to report the error messages you get in this discussion. We'll be glad to help.
> BTW, I'm glad to see interfaces to Julia!!! Thanks,

Thanks. Keep in mind that JSBSim interface with Julia is still experimental at the moment so we'd appreciate any feedback about it.
